packages/yourloops/lib/language.ts

```diff
@@ -117,5 +118,24 @@ function t(s: string, p?: TOptions | string): string {
   return i18n.t(`yourloops|${s}`, p);
 }
 
-export { init, t };
+const getCurrentLocaleName = (i18n: i18next): string => {
+  const shortLocale = i18n.language.split("-")[0] as "en" | "de" | "es" | "fr" | "it" | "nl";
```

Comment:
I would have not prefer to have this list here.
Changing `locales/languages.json` should be the only place where it needs to be.